benchmark/test/reference/multi_vector_distributed.profile.stderr

```diff
@@ -130,3 +128,12 @@ DEBUG: end   dense::scale
 DEBUG: end   repetition
 DEBUG: end   scal
 DEBUG: end   n = 100 
+--------------------------------------------------------------------------
+A system call failed during shared memory initialization that should
+not have.  It is likely that your MPI job will now either abort or
+experience performance degradation.
+
+  Local host:  Tobiass-MacBook-Pro.local
+  System call: unlink(2) /var/folders/q1/_stz6b_s4hb8v5ml97r9vtzc0000gn/T//ompi.Tobiass-MacBook-Pro.501/pid.74813/1/vader_segment.Tobiass-MacBook-Pro.501.60ba0001.2
+  Error:       No such file or directory (errno 2)
+--------------------------------------------------------------------------
```

Comment:
looks like you had a problem regenerating the log

Reply:
haha oops, thanks for letting me know!